ledger/object/index_db_test.go

```diff
@@ -182,6 +184,63 @@ func TestDBIndexStorage_ForID(t *testing.T) {
 	})
 }
 
+func TestDBIndexStorage_ForPulse(t *testing.T) {
+	t.Parallel()
+
+	ctx := inslogger.TestContext(t)
+	pn := gen.PulseNumber()
+
+	t.Run("empty index storage", func(t *testing.T) {
+		t.Parallel()
+
+		tmpdir, err := ioutil.TempDir("", "bdb-test-")
+		defer os.RemoveAll(tmpdir)
+		require.NoError(t, err)
+
+		db, err := store.NewBadgerDB(tmpdir)
+		require.NoError(t, err)
+		defer db.Stop(context.Background())
+		storage := NewIndexDB(db, nil)
+
+		indexes, err := storage.ForPulse(ctx, pn)
+		require.NoError(t, err)
+		require.Equal(t, []record.Index{}, indexes)
+	})
+
+	t.Run("index storage with couple values", func(t *testing.T) {
+		t.Parallel()
+
+		tmpdir, err := ioutil.TempDir("", "bdb-test-")
+		defer os.RemoveAll(tmpdir)
+		require.NoError(t, err)
+
+		db, err := store.NewBadgerDB(tmpdir)
+		require.NoError(t, err)
+		defer db.Stop(context.Background())
+		storage := NewIndexDB(db, nil)
+
+		var indexes []record.Index
+		for i := 0; i < 5; i++ {
+			indexes = append(indexes, record.Index{ObjID: gen.ID()})
+			err = storage.SetIndex(ctx, pn, indexes[i])
+			require.NoError(t, err)
+		}
+
+		realIndexes, err := storage.ForPulse(ctx, pn)
+		require.NoError(t, err)
+		require.Equal(t, 5, len(indexes))
+
+		sort.Slice(indexes, func(i, j int) bool {
```

Comment:
Don't you sort realIndexes, since Badger return it in sorted order?

Reply:
i will sort both